transhumandesign/kag-base
# [Added] Drill heat indicator

## Status

**READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Implement a drill heat indicator like mentioned in issue #383.
the drill heat bar is only visible when hovered over with the mouse when the player is within the (configurable) visibility range of the heat indicator, or when the player is holding the drill.

The time it takes for a drill to cool down has also been lowered.

## Steps to Test or Reproduce

1. Start a game.
2. Buy/Spawn a drill.
3. Hover over the drill, or pick it up.
4. A green bar should now appear.
5. When the drill is used, its bar gets filled up with red, when the red bar reaches 100%, the drill will be dropped like usual.

### Branch used
The branch I used is called 'drill_heat_indication'

### Affected files/objects
[Modified] Drill.as
[New] HeatBar.png

### Screenshots
![preview](https://user-images.githubusercontent.com/22031260/57193365-844dfa80-6f3a-11e9-962b-cf508a38eb1f.gif)



#### Old screenshots
https://imgur.com/a/55OR8hV
